stencil.config.ts

```diff
@@ -24,6 +25,11 @@ export const config: Config = {
     slotChildNodesFix: true,
   },
   outputTargets: [
+    reactOutputTarget({
+      componentCorePackage: '../../dist/types',
```

Comment:
This is named in order to satisfy a generated import, and because we are not using a monorepo. If we set the value to `component-library`, this is the error we get when we compile the bindings:

![image](https://user-images.githubusercontent.com/2008881/128556774-f65b351e-b6e8-414c-9ebe-e733f6b3f41b.png)

The built file `dist/types/index.d.ts` has the export we need:

![image](https://user-images.githubusercontent.com/2008881/128556963-c18c80fc-3410-4aae-a97c-f7a688875bd9.png)